src/clj/kixi/hecuba/data/tariff_calculations.clj

```diff
@@ -0,0 +1,99 @@
+(ns kixi.hecuba.data.tariff-calculations
```

Comment:
I think this should be kixi.hecuba.data.measurements.calculations as they are all generic calculations on measurements. I think kixi.hecuba.data.calculate should be refactored eventually too, but that can be left for now.